tools/restatectl/src/commands/metadata/mod.rs

```diff
@@ -122,3 +136,138 @@ pub async fn create_metadata_store_client(
         .await
         .map_err(|e| anyhow::anyhow!("Failed to create metadata store client: {}", e))
 }
+
+pub async fn get_value<K, T>(opts: &MetadataCommonOpts, key: K) -> anyhow::Result<Option<T>>
```

Comment:
Thanks for reorganizing the code a bit! <3 I was wondering what's better style, to keep `mods.rs` purely about collecting and re-exporting other mods, or to have mixed module-level code in it.